install/helm/gloo/generate/values.go

```diff
@@ -27,12 +27,9 @@ type Global struct {
 	Image      *Image      `json:"image,omitempty"`
 	Extensions interface{} `json:"extensions,omitempty"`
 	GlooRbac   *Rbac       `json:"glooRbac,omitempty"`
-	Wasm       Wasm        `json:"wasm"`
-	GlooStats  Stats       `json:"glooStats" desc:"Config used as the default values for Prometheus stats published from Gloo pods. Can be overridden by individual deployments"`
-}
-
-type Stats struct {
-	Enabled bool `json:"enabled,omitempty" desc:"Controls whether or not prometheus stats are enabled"`
+	Wasm       Wasm        `json:"wasm,omitempty"`
+	GlooStats  Stats       `json:"glooStats,omitempty" desc:"Config used as the default values for Prometheus stats published from Gloo pods. Can be overridden by individual deployments"`
+	GlooMtls   Mtls        `json:"glooMtls,omitempty" desc:"Config used to enable internal mtls authentication"`
```

Comment:
let's be more specific, perhaps
```suggestion
	GlooMtls   Mtls        `json:"glooMtls,omitempty" desc:"Config used to enable internal mtls authentication (currently just Gloo to Envoy communication)"`
```